tests/integration/interactive/cmd_init_test.go

```diff
@@ -59,4 +61,126 @@ var _ = Describe("odo init interactive command tests", func() {
 		Expect(output).To(ContainSubstring("Your new component \"my-go-app\" is ready in the current directory."))
 		Expect(helper.ListFilesInDir(commonVar.Context)).To(ContainElements("devfile.yaml"))
 	})
+
+	Describe("Displaying welcoming messages", func() {
+
+		type tester func(*expect.Console, *bytes.Buffer)
```

Comment:
Let's describe this type in `helper_interactive.go` and use it in the definition of `func RunInteractive`, so it can be reused